assets/airports/eidw.json

```diff
@@ -883,124 +1076,361 @@
+                "28X": [["NIMAT", "A140|S250"], ["KERAV", "A80|S230"], "KOGAX", "KUDOM", "DW814", "DW815", ["DW816", "A80"], ["NARMU", "A30|S200"], ["LAPMO", "A30|S180"]],
+                "16X": [["NIMAT", "A120"], ["KERAV", "A60"]],
+                "34X": [["NIMAT", "A120"], ["KERAV", "A60"]]
+            }
+        },
+        "BOYNE": {
+            "icao": "BOYNE",
+            "name": "Boyne",
+            "entryPoints": {
+                "EGNS": ["EGNS", ["IOM", "A200-"]]
+            },
+            "rwy": {
+                "10": [["BOYNE", "A180"], ["EKREN", "A140"], ["ELBON", "A100"], ["ASDER", "A80|S230"], "AKIVA", "ADNAL", "APRUT", "DW865", ["DW866", "A80"], ["NEKIL", "A40"]],
+                "28": [["BOYNE", "A110"], ["ADSIS", "A90"], ["KERAV", "A80|S230"], "KOGAX", "KUDOM", "DW814", "DW815", ["DW816", "A80"], ["NARMU", "A30|S200"], ["LAPMO", "A30|S180"]],
+                "16": [["BOYNE", "A90"], ["ADSIS", "A70"], ["KERAV", "A60"]],
+                "34": [["BOYNE", "A90"], ["ADSIS", "A70"], ["KERAV", "A60"]],
+                "10X": [["BOYNE", "A180"], ["EKREN", "A140"], ["ELBON", "A100"], ["ASDER", "A80|S230"], "AKIVA", "ADNAL", "APRUT", "DW865", ["DW866", "A80"], ["NEKIL", "A40"]],
+                "28X": [["BOYNE", "A110"], ["ADSIS", "A90"], ["KERAV", "A80|S230"], "KOGAX", "KUDOM", "DW814", "DW815", ["DW816", "A80"], ["NARMU", "A30|S200"], ["LAPMO", "A30|S180"]],
+                "16X": [["BOYNE", "A90"], ["ADSIS", "A70"], ["KERAV", "A60"]],
+                "34X": [["BOYNE", "A90"], ["ADSIS", "A70"], ["KERAV", "A60"]]
+            }
+        },
+        "BAGSO": {
+            "icao": "BAGSO",
+            "name": "Bagso",
+            "entryPoints": {
+                "RAMOX": [["RAMOX", "A170|S300"], ["OGTUG", "A140|S250"]]
+            },
+            "rwy": {
+                "10": [["BAGSO", "A170"], ["EKREN", "A140"], ["ELBON", "A100"], ["ASDER", "A80|S230"], "AKIVA", "ADNAL", "APRUT", "DW865", ["DW866", "A80"], ["NEKIL", "A40"]],
+                "28": [["BAGSO", "A100"], ["ADSIS", "A90"], ["KERAV", "A80|S230"], "KOGAX", "KUDOM", "DW814", "DW815", ["DW816", "A80"], ["NARMU", "A30|S200"], ["LAPMO", "A30|S180"]],
+                "16": [["BAGSO", "A90"], ["ADSIS", "A70"], ["KERAV", "A60"]],
+                "34": [["BAGSO", "A90"], ["ADSIS", "A70"], ["KERAV", "A60"]],
+                "10X": [["BAGSO", "A170"], ["EKREN", "A140"], ["ELBON", "A100"], ["ASDER", "A80|S230"], "AKIVA", "ADNAL", "APRUT", "DW865", ["DW866", "A80"], ["NEKIL", "A40"]],
+                "28X": [["BAGSO", "A100"], ["ADSIS", "A90"], ["KERAV", "A80|S230"], "KOGAX", "KUDOM", "DW814", "DW815", ["DW816", "A80"], ["NARMU", "A30|S200"], ["LAPMO", "A30|S180"]],
+                "16X": [["BAGSO", "A90"], ["ADSIS", "A70"], ["KERAV", "A60"]],
+                "34X": [["BAGSO", "A90"], ["ADSIS", "A70"], ["KERAV", "A60"]]
+            }
+        },
+        "LIPGO": {
```

Comment:
I show `LIPGO` as not being its own procedure, but rather a waypoint along the `ABLI1L`. Do you concur?